### PR TITLE
Don't quote generic font families

### PR DESCRIPTION
**Pull request by hterkelsen**
This is a change that should have been upstreamed before.

### CODE REVIEW

File: lib/web_ui/lib/src/engine/util.dart [hunk 1]
```diff
@@ -238,3 +238,30 @@ String _pathToSvgClipPath(ui.Path path,
...
 }
+
+/// From: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#Syntax
+///
+/// Generic font families are a fallback mechanism, a means of preserving some
+/// of the style sheet author's intent when none of the specified fonts are
+/// available. Generic family names are keywords and must not be quoted. A
+/// generic font family should be the last item in the list of font family
+/// names.
+const List<String> _genericFontFamilies = <String>[
```
Comment (jonahwilliams) [inline lib/web_ui/lib/src/engine/util.dart]: Could this be a Set?

Comment (hterkelsen) [reply lib/web_ui/lib/src/engine/util.dart]: Yes.

**Comment by hterkelsen** [comment]
Added a test

**Comment by ditman** [comment]
Thanks for fixing this! 

_(I've just fixed my github notifications so I can see these mentions in the future!)_

**Pull request merged**